doctrine/event_listeners_subscribers.rst

```diff
@@ -169,8 +169,8 @@ interface and have an event method for each event it subscribes to::
     namespace AppBundle\EventListener;
 
     use Doctrine\Common\EventSubscriber;
-    use Doctrine\ORM\Event\LifecycleEventArgs;
-    // for Doctrine 2.4: Doctrine\Common\Persistence\Event\LifecycleEventArgs;
+    // for Doctrine < 2.4: use Doctrine\ORM\Event\LifecycleEventArgs;
```

Comment:
We can't replace "Doctrine 2.4" by "Doctrine < 2.4". It should be "Doctrine <= 2.4" or "Doctrine < 2.5", right?